clients/python/requirements/test.txt

```diff
@@ -5,3 +5,10 @@ pytest
 papermill
 ipython
 ipykernel
+pandas
```

Comment:
MINOR: keep all listing alphabetical. It will help you to spot typos and duplicates

Reply:
done